src/Chainweb/Pact/PactService.hs

```diff
@@ -1071,6 +1072,10 @@ setBlockData bh = do
 playOneBlock
     :: (PayloadCas cas)
     => BlockHeader
+        -- ^ this is the current header. We may consider changing this to the parent
```

Comment:
See review https://github.com/kadena-io/chainweb-node/pull/902#pullrequestreview-362793443, we should change this now (review usages of `currHeader` below to switch to using parent header.

Reply:
There is an upcoming PR that addresses the usage of `currHeader` vs. `parentHeader` more in depth. Let's revisit this in the review of that PR.